<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LidarProcessingDialog</class>
 <widget class="QDialog" name="LidarProcessingDialog">
  <property name="windowTitle">
   <string>LiDAR Processing Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelInputFile">
       <property name="text">
        <string>Input LAS/LAZ File:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="lineEditInputFile"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowseInput">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelOutputFile">
       <property name="text">
        <string>Output DEM File:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="lineEditOutputFile"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowseOutput">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelScalar">
       <property name="text">
        <string>Scalar:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spinScalar">
       <property name="value">
        <double>1.2</double>
       </property>
       <property name="singleStep">
        <double>0.1</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSlope">
       <property name="text">
        <string>Slope:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="spinSlope">
       <property name="value">
        <double>0.2</double>
       </property>
       <property name="singleStep">
        <double>0.05</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelThreshold">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="spinThreshold">
       <property name="value">
        <double>0.45</double>
       </property>
       <property name="singleStep">
        <double>0.05</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelWindow">
       <property name="text">
        <string>Window:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="spinWindow">
       <property name="value">
        <double>16.0</double>
       </property>
       <property name="singleStep">
        <double>1.0</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelResolution">
       <property name="text">
        <string>DEM Resolution:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="spinResolution">
       <property name="value">
        <double>1.0</double>
       </property>
       <property name="singleStep">
        <double>0.5</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox.Cancel|QDialogButtonBox.Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
